templates/contribute.html

```diff
@@ -0,0 +1,109 @@
+{% extends 'global/base.html' %}
+{% load staticfiles %}
+
+{% block content %}
+
+    <div class="container">
+
+        <section id="resources">
+            <div class="row">
+                <div class="col-md-12">
+                    <h2>Contribute</h2>
+                    <p>
+                        There are many ways to contribute to Django Girls: you can organize an event, coach, work on the tutorial and help on Gitter. If you don't have time but want to help, you can always support us on <a href="https://www.patreon.com/djangogirls">Patreon</a>.
+                    </p>
+                </div>
+            </div>
+
+            <div class="row clearfix">
+                <div class="col-md-6">
+                    <h3>Organize!</h3>
+                    <div class="cover">
+                        <a href="http://organize.djangogirls.org/">
+                            <img src="{% static 'img/global/contribute/organize.jpg' %}" alt="Flowers and name tag on a table." />
+                        </a>
+                    </div>
+                    <div class="info">
+
+                        <p>Each Django Girls event is a free, one-day workshop for 30-60 people on HTML, CSS, Python and Django. A typical workshop takes 8 hours and people work in small, 3-person groups with a coach dedicated to each group.</p>
+
+                        <p>Want to get started? Check our <a href="http://organize.djangogirls.org/">Organizer's Manual</a> and <a href="https://djangogirls.org/organize/">bring Django Girls to your city!</a></p>
+                    </div>
+                </div>
+                <div class="col-md-6">
+                    <h3>Coach!</h3>
+                    <div class="cover">
+                        <a href="http://djangogirls.org/events">
+                            <img src="{% static 'img/global/contribute/coach.jpg' %}" alt="A coach helping an attendee." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>You have just heard that an event is happening in your city and went to help? Contact the local organizing team and ask if they need more coaches!</p>
```

Comment:
Change to "Have you heard about an event happening in your city and you want to help?"